src/server/graphql/RateLimiter.js

```diff
@@ -0,0 +1,55 @@
+import ms from 'ms'
+
+const HOUR = ms('1h')
+const MINUTE = ms('1m')
+
+class RateLimiter {
+  constructor () {
+    this._records = {}
+    this._lastCall = {}
+    setInterval(() => {
+      this.gc()
```

Comment:
Is there a reason that you want to trigger the gc manually? Does calling gc on an object do something special? I did a quick google to try and figure out how this was different than calling the global gc but it must be new enough that it evaded my first page results...